sqrl-server/sqrl-server-vertx/src/main/java/com/datasqrl/graphql/GraphQLServer.java

```diff
@@ -222,14 +249,14 @@ private void addHealthCheckHandler(Router router) {
   }
 
   private Optional<JWTAuthHandler> createJwtAuthHandler() {
-    if (this.config.getJWTAuthOptions() != null) {
-      JWTAuth provider = JWTAuth.create(vertx, config.getJWTAuthOptions());
-      return Optional.of(JWTAuthHandler.create(provider));
+    if (config.getJWTAuthOptions() != null) {
+      authProvider = JWTAuth.create(vertx, config.getJWTAuthOptions());
```

Comment:
I know it is not a priority, but this is a good example why using a dependency injection framework could be beneficial. Could we consider adopting one in the long term?